src/main/java/com/yahoo/bullet/querying/Querier.java

```diff
@@ -366,6 +374,10 @@ public Querier(Mode mode, RunningQuery query, BulletConfig config) {
     @Override
     @SuppressWarnings("unchecked")
     public Optional<List<BulletError>> initialize() {
+        int maxCount = config.getAs(BulletConfig.RATE_LIMIT_MAX_EMIT_COUNT, Integer.class);
+        int interval = config.getAs(BulletConfig.RATE_LIMIT_TIME_INTERVAL, Integer.class);
+        maxRateLimit = (maxCount / (double) interval) * RateLimiter.SECOND;
```

Comment:
This is already created and initialized below in the if branch. Should we just make RateLimiter take the config and extract this info and store it?

Reply:
Done. Use absoluteRateLimit in RateLimiter instead.